content/04.results.md

```diff
@@ -0,0 +1,16 @@
+# Results
+
+1. Figure 1 - Visual to show that alignment works along intra year variation (Procrustes Validation)
+  1. Umap panel for an individual year to show models are separated
+  2. Umap panel to show the same year after models have been aligned
+  3. Umap panel to show across years (inter-year varation)
+  	  1. might not have this as a figure given the aligned umap results, but we shall see 
+2. Figure 2 - CUSUM validation
+  1. Table of found timepoint changes using this algorithm
+  2. Highlight pandemic as positive control
+  3. Nearest Neighbors upset plot ^^
+  4. Might look into lung cancer or other form of cancer results
+4. Figure 3 - Website walkthrough for the work done here
+  1. Website screenshots
```

Comment:
The figures that make up the website might be nice to show/explain individually - not as screenshots but as actual figures (remember the PLOS situation where using a screenshot was an issue we had to work through).